CONTRIBUTING.rst

```diff
@@ -151,8 +151,8 @@ but here is a simple overview:
 
     $ git clone git@github.com:YOUR_GITHUB_USERNAME/pytest.git
     $ cd pytest
-    $ git checkout pytest-2.7      # if you want to fix a bug for the pytest-2.7 series
-    $ git checkout master          # if you want to add a feature bound for the next minor release
+    $ git checkout 2.8.X   # if you want to fix a bug for the pytest-2.8 series
```

Comment:
I remember you mentioned in the mailing list about changing the meaning of the branches, so we have `master` which contains code for the releases, and have a `2.9` branch which contain only new features that should go in the next minor version.

The reasoning is that most contributions from outside users is for bugs and minor fixes, makes sense using the default target branch being set correctly by default when creating a PR.